guide/13-managing-arcgis-applications/hub-submodule-intro.ipynb

```diff
@@ -0,0 +1,71 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6cec2944",
+   "metadata": {},
+   "source": [
+    "## The `hub` submodule\n",
+    "\n",
+    "The `hub` submodule within `arcgis.apps` acts as the pythonic interface to [ArcGIS Hub](https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources). It enables automation of several Hub workflows and simplifies the use of the Hub information model by allowing access to Hub objects such as `Initiative`, `Event`, `Site` and `Page`.\n",
```

Comment:
* I think a link to the [Overview](https://www.esri.com/en-us/arcgis/products/arcgis-hub/overview) would serve better in the context of this sentence. Followed perhaps by another sentence introducing the resources and linking it to the resources.
  * _For tutorials and additional resources on using ArcGIS Hub, see [here](https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources)_